src/server/templates/register-form/summary.njk

```diff
@@ -101,7 +101,7 @@
       <button type="submit" id="sendBtn" class="button">Send my details</button>
     </form>
 
-    <p>
+    <p class="summeryNote">
```

Comment:
Shouldn't need a specific classname that is tied heavily to its contents to apply spacing. This was effectively handled using `* + *` previously.